### ISSUE TITLE
GraphQL Reference page service is 503

### ISSUE THREAD

**Issue by ntloi95** [gatsbyjs/gatsby#7509]
GraphQL Reference page https://next.gatsbyjs.org/docs/graphql-reference/
<img width="736" alt="screen shot 2018-08-21 at 3 06 08 pm" src="https://user-images.githubusercontent.com/9493402/44388964-c4f41800-a553-11e8-9997-d9fe4e4aa51c.png">

**Comment by Chuloo** [comment]
Works on my end. How did you arrive at that? 😄

**Comment by ntloi95** [comment]
I learn GraphQL with gatsby. But the demo is 503.

**Comment by eric-personal** [comment]
Yep same here, 503 showing up on this link https://next.gatsbyjs.org/docs/graphql-reference/

**Comment by azdanov** [comment]
Works for me. 🤷‍♂️

**Comment by jendowns** [comment]
👋 Hey there, I'm also seeing 503 errors for all images on this page as well: https://next.gatsbyjs.org/docs/graphql-reference/ (in Firefox 61 and Chrome 68)

And for the record, I get 503 errors for the v1 version as well: https://www.gatsbyjs.org/docs/graphql-reference/

**Is there any way these examples could be put in `<code>` blocks instead?** 😇

**Comment by m-allanson** [comment]
These should be working now. They're live demos, but the way they're put together is a bit of a hack (which is my fault 😅)... normally when the demo app breaks it will restart itself, but it can take a few minutes for it to come back online. 

Text-based fallbacks sound like a good idea, but ideally the demo would 'just' work all the time. I've added an uptime check to the demo to get an idea of how reliable it is.

In the meantime please report here if you see it's not working!
